internal/server/server.go

```diff
@@ -200,7 +200,7 @@ func New(options Options) (*Server, error) {
 	}
 
 	if options.EnableTelemetry {
-		server.tel = NewTelemetry(server.db, db.DefaultOrgSettings.ID)
+		server.tel = NewTelemetry(server.db, db.DefaultOrg.ID)
```

Comment:
This might be a problem, the `DefaultOrg.ID` is always `1000`, so this is going to make every install look like the same one.  I think we need a different ID here.

Maybe we can keep the `settings.ID` stored as a field on `Organization`, something like `InstallID` ? For new installs we can generate a new `uid.ID` for them, so they all have different IDs.

Reply:
Good catch!